README.md

```diff
@@ -2,6 +2,7 @@
 
 Cartopy
 -------
+[![Build Status](https://api.travis-ci.org/repositories/SciTools/cartopy.svg?branch=master)](http://travis-ci.org/SciTools/cartopy/branches)
```

Comment:
Yes, one less http link is one less link that can be MITM'd and one less request for the user that clicks it.